src/ReleaseHistory.md

```diff
@@ -263,5 +263,8 @@
 * BUGFIX: 'region.charOffset' default value should be -1 (invalid value) rather than 0. Fixes an issue where 'region.charLength' is > 0 but 'region.charOffset' is absent (because its value of 0 was incorrectly elided due to being the default value). 
 
 ## **v2.0.0-csd.2.beta.2019.02.20** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.0.0-csd.2.beta.2019.02.20) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.0.0-csd.2.beta.2019.02.20) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.0.0-csd.2.beta.2019.02.20)) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.0.0-csd.2.beta.2019.02.20))
-* API BREAKING: Remove 'invocation'attachments', now replaced by 'run.tool.extensions'. https://github.com/oasis-tcs/sarif-spec/issues/327
+* API BREAKING: Remove 'invocation.attachments', now replaced by 'run.tool.extensions'. https://github.com/oasis-tcs/sarif-spec/issues/327
+
+## **v2.0.0-csd.2.beta.2019.02.20** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.0.0-csd.2.beta.2019.02.20) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.0.0-csd.2.beta.2019.02.20) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.0.0-csd.2.beta.2019.02.20)) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.0.0-csd.2.beta.2019.02.20))
```

Comment:
>v2.0.0-csd.2.beta.2019.02.20* [](start = 5, length = 29)

Don't recap this header. :)

Reply:
oops! :) thanks!

---
In reply to: [259556888](https://github.com/Microsoft/sarif-sdk/pull/1306#discussion_r259556888) [](ancestors = 259556888)